clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ActiveActiveStoreIngestionTask.java

```diff
@@ -66,13 +69,24 @@
  */
 public class ActiveActiveStoreIngestionTask extends LeaderFollowerStoreIngestionTask {
   private static final Logger LOGGER = LogManager.getLogger(ActiveActiveStoreIngestionTask.class);
+  private static final byte[] BINARY_DECODER_PARAM = new byte[16];
+
   private final int rmdProtocolVersionID;
   private final MergeConflictResolver mergeConflictResolver;
   private final RmdSerDe rmdSerDe;
   private final Lazy<KeyLevelLocksManager> keyLevelLocksManager;
   private final AggVersionedIngestionStats aggVersionedIngestionStats;
   private final RemoteIngestionRepairService remoteIngestionRepairService;
 
+  private static class ReusableObjects extends AvroSerializer.AvroSerializerReusableObjects {
```

Comment:
I think it doesn't need to extend that other class, since we don't use anything it contains, right?

Reply:
right, copy pasta extend :(